org.eclipse.xtext.dev-bom/pom.xml

```diff
@@ -0,0 +1,482 @@
+			<email>sven.efftinge@typefox.io</email>
+			<organization>TypeFox</organization>
+			<organizationUrl>http://typefox.io</organizationUrl>
+		</developer>
+		<developer>
+			<name>Sebastian Benz</name>
+		</developer>
+		<developer>
+			<name>Lorenzo Bettini</name>
+			<email>lorenzo.bettini@gmail.com</email>
+			<organization>DISIA, University Firenze</organization>
+		</developer>
+		<developer>
+			<name>Michael Clay</name>
+		</developer>
+		<developer>
+			<name>Arne Deutsch</name>
+			<email>arne.deutsch@itemis.de</email>
+			<organization>itemis</organization>
+			<organizationUrl>http://www.itemis.com</organizationUrl>
+		</developer>
+		<developer>
+			<name>Christian Dietrich</name>
+			<email>christian.dietrich@itemis.de</email>
+			<organization>itemis</organization>
+			<organizationUrl>http://www.itemis.com</organizationUrl>
+		</developer>
+		<developer>
+			<name>Moritz Eysholdt</name>
+			<email>moritz.eysholdt@typefox.io</email>
+			<organization>TypeFox</organization>
+			<organizationUrl>http://typefox.io</organizationUrl>
+		</developer>
+		<developer>
+			<name>Dennis Hübner</name>
+			<email>dennis.huebner@typefox.io</email>
+			<organization>TypeFox</organization>
+			<organizationUrl>http://typefox.io</organizationUrl>
+		</developer>
+		<developer>
+			<name>Jan Köhnlein</name>
+			<email>jan.koehnlein@typefox.io</email>
+			<organization>TypeFox</organization>
+			<organizationUrl>http://typefox.io</organizationUrl>
+		</developer>
+		<developer>
+			<name>Anton Kosyakov</name>
+			<email>anton.kosyakov@typefox.io</email>
+			<organization>TypeFox</organization>
+			<organizationUrl>http://typefox.io</organizationUrl>
+		</developer>
+		<developer>
+			<name>Tamas Miklossy</name>
+			<email>miklossy@itemis.de</email>
+			<organization>itemis</organization>
+			<organizationUrl>http://www.itemis.com</organizationUrl>
+		</developer>
+		<developer>
+			<name>Stefan Oehme</name>
+		</developer>
+		<developer>
+			<name>Holger Schill</name>
+			<email>holger.schill@itemis.de</email>
+			<organization>itemis</organization>
+			<organizationUrl>http://www.itemis.com</organizationUrl>
+		</developer>
+		<developer>
+			<name>Christian Schneider</name>
+			<email>christian.schneider@typefox.io</email>
+			<organization>TypeFox</organization>
+			<organizationUrl>http://typefox.io</organizationUrl>
+		</developer>
+		<developer>
+			<name>Miro Spönemann</name>
+			<email>miro.spoenemann@typefox.io</email>
+			<organization>TypeFox</organization>
+			<organizationUrl>http://typefox.io</organizationUrl>
+		</developer>
+		<developer>
+			<name>Karsten Thoms</name>
+			<email>karsten.thoms@itemis.de</email>
+			<organization>itemis</organization>
+			<organizationUrl>http://www.itemis.com</organizationUrl>
+		</developer>
+		<developer>
+			<name>Knut Wannheden</name>
+		</developer>
+		<developer>
+			<name>Sebastian Zarnekow</name>
+			<email>sebastian.zarnekow@gmail.com</email>
+			<organization>itemis</organization>
```

Comment:
I'm not affiliated with itemis at this time. Please remove the organization information.